src/auth/jwt-error.ts

```diff
@@ -0,0 +1,22 @@
+import express, { NextFunction } from "express";
+import { JwtError } from "@controllers/public-rest";
+
+// Handle TSOA errors
+export function jwtErrorHandler(
+  error: Error,
+  req: express.Request,
+  res: express.Response,
+  next: NextFunction
+) {
+  console.log("errorHandler::error", error);
+
+  if (isJwtError(error) && error.status !== 200) {
+    return res.status(401).json(error);
+  } else {
+    next();
+  }
+}
+
+function isJwtError(err: Error): err is JwtError {
+  return (err as JwtError).status > 0 && "message" in err;
```

Comment:
I'm afraid this check will also trigger with other kind of errors since a status and message is fairly common in error objects.
Is there no stricter way to check for JwtError? if JwtError is a class, then we can do an `instanceof` instead.

Reply:
Comment is obsolete if we use the existing jwt files